ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -222,28 +222,34 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-sno-etcd-encryption-arm-f2
-  cron: 24 16 13,18 * *
+- as: aws-ipi-sno-etcd-encryption-basecap-none-arm-f2
```

Comment:
I suggest to update `f2` to `f14` per the original cron setting, to save some cost.

Reply:
updated